.github/workflows/create-tag.yaml

```
@@ -31,7 +31,7 @@ jobs:
         uses: negz/create-tag@v1
         with:
           version: ${{ github.event.inputs.version }}
-          token: ${{ secrets.GITHUB_TOKEN }}
+          token: ${{ secrets.USER_GITHUB_TOKEN }}
```

@travisn can you please add a personal token with this name or let me know if you have suggestions
```suggestion
          token: ${{ secrets.USER_GITHUB_TOKEN }}
```

---

A user secret would be added to their own fork, but the action is being run against the main fork, right? I'm not seeing how this would help.